src/labels/CustomLabels.labels

```diff
@@ -7528,33 +7528,34 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <categories>Gift Entry</categories>
         <language>en_US</language>
         <protected>true</protected>
-        <shortDescription>Failure msg description for system error &amp; uncertain if card charged - Part 1</shortDescription>
+        <shortDescription>Failure msg description for system error &amp; uncertain if payment processed - Part 1</shortDescription>
         <!-- {0} = Donation Amount {1} = Donor Name {2} = custom label commonPaymentServices i.e. Payment Services -->
-        <value>A system error occurred for the {0} donation by {1}. Your admin should review transactions in {2} and determine if the card was charged.</value>
+        <value>A system error occurred for the {0} donation by {1}. Your admin should review transactions in {2} and determine if the payment was processed.</value>
     </labels>
     <labels>
         <fullName>geErrorUncertainCardChargePart2</fullName>
         <categories>Gift Entry</categories>
         <language>en_US</language>
         <protected>true</protected>
-        <shortDescription>Failure msg description for system error &amp; uncertain if card charged - Part 2</shortDescription>
-        <value>If the card was not charged, reenter the gift and card payment in Gift Entry.</value>
+        <shortDescription>Failure msg description for system error &amp; uncertain if payment processed - Part 2</shortDescription>
+        <value>If the payment was not processed, reenter the gift and payment information in Gift Entry.</value>
     </labels>
     <labels>
         <fullName>geErrorUncertainCardChargePart3</fullName>
         <categories>Gift Entry</categories>
         <language>en_US</language>
         <protected>true</protected>
-        <shortDescription>Failure msg description for system error &amp; uncertain if card charged - Part 3</shortDescription>
+        <shortDescription>Failure msg description for system error &amp; uncertain if payment processed - Part 3</shortDescription>
         <!-- {0} = custom label commonPaymentServices i.e. Payment Services -->
-        <value>If the card was charged, note the Transaction ID in {0}, and search for records with that ID in Salesforce. Update the gift and donor records as needed.</value>
+        <value>If the payment was processed, note the Transaction ID in {0}, and search for records with that ID in Salesforce. Update the gift and donor records as needed.</value>
     </labels>
     <labels>
         <fullName>geErrorUncertainCardChargePart4</fullName>
         <categories>Gift Entry</categories>
         <language>en_US</language>
         <protected>true</protected>
-        <shortDescription>Failure msg description for system error &amp; uncertain if card charged - Part 4</shortDescription>
+        <shortDescription>Failure msg  description for system error &amp; uncertain if card charged - Part
```

Comment:
```suggestion
        <shortDescription>Failure msg description for system error &amp; uncertain if card processed - Pt 4
```

Reply:
@melissabarber do we want to say `payment processed` on this one instead of `card processed`?